integrations/server/test_covidcast_meta.py

```diff
@@ -33,15 +33,18 @@ class CovidcastMetaTests(unittest.TestCase):
   }
 
   template = '''
-      INSERT INTO
-        `signal_latest` (`signal_data_id`, `signal_key_id`, `geo_key_id`,
-	      `time_type`, `time_value`, `value_updated_timestamp`,
+      INSERT INTO `signal_latest` (
+        `signal_data_id`, `signal_key_id`, `geo_key_id`,
+        `time_type`, `time_value`, `value_updated_timestamp`,
         `value`, `stderr`, `sample_size`,
         `issue`, `lag`, `missing_value`,
         `missing_stderr`,`missing_sample_size`)
       VALUES
-        (%d, %d, %d, "%s", %d, 123,
-        %d, 0, 0, %d, 0, %d, %d, %d)
+        (%d, %d, %d,
```

Comment:
Do we have code standards set for line length and how to format things like this? Seems to change from file to file and it would be nice to have a unified truth for the project.

Reply:
Not in this repo, and we should! Outside the scope of this PR but [I made an issue for it. ](https://github.com/cmu-delphi/delphi-epidata/issues/962)

Reply:
we dont have any hard rules.  generally, imnsho, the format should be something that "looks clean" (or at least is not sloppy), preferably with a structure that makes it easy to read, identifies/highlights important parts, and otherwise means something semantically (whenever possible).

this section you highlighted includes sql with column names that need to follow the same order as the values provided shortly after; thus the formatting here has line breaks that match up 1:1, so that each row of column names corresponds to a later row for its values.  that way, its easy to see (for example) that the last item of the second row of column names (`value_updated_timestamp`) refers to the last item of the second row of values (`123`).

all this test code has come a long way, but there is still more room for improvement...  see #897 

for this pr, i tried not to change anything where i didnt have to, to keep the [diff between this and source material](https://github.com/cmu-delphi/delphi-epidata/compare/fbf878e..0d4a2e7) as small as possible.  the intent was to preserve the spirit of the test refactoring & improvements from the other branch without complicating it by introducing other changes.  however, i goofed a couple times and introduced some whitespace changes but ill claim they were for the best ¯\\\_(ツ)_/¯